corehq/pillows/xform.py

```diff
@@ -214,8 +212,9 @@ def get_xform_pillow(pillow_id='xform-pillow', ucr_division=None,
     )
     unknown_user_form_processor = UnknownUsersProcessor()
     form_meta_processor = FormSubmissionMetadataTrackerProcessor()
-    checkpoint_id = "{}-{}-{}-{}".format(
-        pillow_id, XFORM_INDEX_INFO.index, REPORT_XFORM_INDEX_INFO.index, USER_INDEX)
+    # TODO Do we need to preserve this checkpoint format?
+    checkpoint_id = "{}-{}-report_xforms_20160824_1708-{}".format(
```

Comment:
nit: remove the duplicate index fragment
```suggestion
    checkpoint_id = "{}-{}-{}".format(
```